style/javascript/.eslintrc.json

```diff
@@ -115,7 +115,7 @@
     ],
     "quotes": [
       2,
-      "single",
+      "double",
```

Comment:
It seems like the JS community mostly uses single quotes. I’d personally prefer that we try to conform to community styles as opposed to trying to be consistent across stacks (i.e. use double quotes for all languages).

Reply:
[This article](https://bytearcher.com/articles/single-or-double-quotes-strings-javascript/) has some statistics on quote usage. It seems like single quotes are more popular, but some prominent style guides (JSLint, ESLint, Typescript) prefer double quotes. I've changed my mind to being OK with this change.